documentatie/gebruikersdocumentatie/installeren-en-starten.md

```diff
@@ -0,0 +1,91 @@
+# Abacus installeren en starten
+
+Het installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
+Hieronder vind je instructies voor de verschillende methodes.
+
+## Linux en macOS
+
+De onderstaande methodes werken waarschijnlijk op alle gangbare (recente) Linux-distributies en macOS, en zijn getest op Ubuntu (22.04 en hoger) en Debian (12/bookworm).
+
+### Methode 1: build artifact downloaden en starten
+
+Deze methode lijkt het meest op het draaien van een productiebuild en is ook het meest eenvoudig. Voer de volgende stappen uit:
+
+1. Ga in de Abacus-repository naar [Actions](https://github.com/kiesraad/abacus/actions)
+2. Klik aan de linkerkant op [Build, lint & test](https://github.com/kiesraad/abacus/actions/workflows/build-lint-test.yml)
+3. Klik aan de rechterkant op *Branch* om op een branch te filteren
+4. Selecteer de meest recente action
+5. Download het bestand *backend-build*
+
+![instructions1](https://github.com/user-attachments/assets/9efb524e-9256-43eb-9d98-141e049c8ba9)
+![instructions2](https://github.com/user-attachments/assets/b8389384-98fa-42c1-905f-81db7e301942)
+
+De download bevat een ZIP-bestand met de binary. Deze binary bevat ook al alle frontend-assets. Pak deze binary uit en start hem als volgt:
+
+```sh
+./path/to/binary/api
+```
```

Comment:
Als ik de applicatie op deze manier start, dan krijg ik de pagina "Verkiezingen niet gevonden". Dat gebeurt niet als ik `--seed-data` toevoeg.

Reply:
Dit geeft alleen voorbeelden hoe je de binary zou kunnen runnen. Het geval wat je hierboven beschrijft wordt behandeld in het volgende voorbeeld. Misschien moet daar de bijbehorende comment wat aangepast worden.

Reply:
Ik heb de volgorde aangepast en uitgelegd dat je geen data krijgt als je geen argumenten gebruikt bij het starten.